recipe/meta.yaml

```
@@ -1,41 +1,54 @@
 {% set name = "annotated-types" %}
+{% set alt_name = name.replace('-','_') %}
 {% set version = "0.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/annotated_types-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ alt_name }}/{{ alt_name }}-{{ version }}.tar.gz
   sha256: 563339e807e53ffd9c267e99fc6d9ea23eb8443c08f112651963e24e22f84a5d
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - hatchling
   run:
-    - python >=3.7
+    - python
     - typing-extensions >=4.0.0
```

```suggestion
    - typing-extensions >=4.0.0 # [py<39]
```
https://github.com/annotated-types/annotated-types/blob/v0.6.0/pyproject.toml#L32